### PR TITLE
Strings reworked 8

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -30,19 +30,19 @@
...
     <string name="use_login_password">Use login and password</string>
     <string name="open_street_map_login_mode">You need to login to upload new or modified changes. \n\nYou can log in using the safe OAuth method or use your login and password.</string>
-    <string name="osm_edits_view_descr">You can view all your not yet uploaded edits or OSM bugs in %1$s. Uploaded points don’t show in OsmAnd.</string>
+    <string name="osm_edits_view_descr">All your not yet uploaded edits or OSM bugs can be viewed in %1$s. Uploaded points don’t show up in OsmAnd.</string>
     <string name="sign_in_with_open_street_map">Sign in with OpenStreetMap</string>
     <string name="login_open_street_map_org">Login to OpenStreetMap.org</string>
     <string name="login_open_street_map">Login to OpenStreetMap</string>
     <string name="plugin_global_prefs_info">These plugin setting are global, and apply to all profiles</string>
-    <string name="message_you_need_add_two_points_to_show_graphs">You need to add at least two points</string>
+    <string name="message_you_need_add_two_points_to_show_graphs">Add at least two waypoints</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:45]: ?

Comment (sonora) [reply OsmAnd/res/values/strings.xml:45]: We use point and waypoint in differing meanings, I think.

Comment (comradekingu) [reply OsmAnd/res/values/strings.xml:45]: If so, what kind of points are "points"?

Comment (sonora) [reply OsmAnd/res/values/strings.xml:45]: `<trkpt>`, I believe.

**Comment by sonora** [comment]
Already merged with some imodifications - thx!

**Pull request closed without merging**